portal-api/src/v1/users/transform-database-user.js

```diff
@@ -1,14 +1,20 @@
+const { produce } = require('immer');
```

Comment:
What is the reason to use immer?
Can we use deep cloning like function `structuredClone`? if now that sub object links we want to preserve?

Reply:
Ah I'll update to this, on further reading immer doesn't do what im after

Reply:
I've updated to lodash clonedeep. This is due to structuredClone not cloning buffers correctly